README.md

```diff
@@ -49,7 +49,8 @@ If you are working with Spartacus 5.x, your Angular development environment shou
 
 - [Angular CLI](https://angular.io/): Version **14.2.3** or newer, < 15.
 - [Node.js](https://nodejs.org/en/): Version **14.15** or newer (but **less than** version 15), or **16.10** or newer.
-- [Yarn](https://classic.yarnpkg.com/lang/en/): Version **1.15** or newer.
+- [Yarn](https://classic.yarnpkg.com/lang/en/): Version **1.15** or newer (**OR** from Spartacus 5.2 [npm](https://www.npmjs.com/): Version **6.14** or newer.)
```

Comment:
Referring to your proposal - Well, I'm not quite sure how to approach this topic yet. We can't just post about `npm` support under section for version 5.x because it won't be true. The version we currently have on the `develop` branch won't install without using the `--legacy-peer-deps` flag due to dependency issues that are being fixed in this PR, so in theory we only support `npm` as of version 5.2